app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
```

Comment:
JavaScript is garbage collected. As soon as no reference to the object exists anymore it will be marked for removal. So a local variable of a function will cease to exist when the function ends.